docs/components/board/_index.md

```diff
@@ -320,7 +323,7 @@ The following properties are available for `i2cs`:
 | Name | Type | Inclusion | Description |
 | ---- | ---- | --------- | ----------- |
 |`name`| string| **Required** | `name` of the I<sup>2</sup>C bus. |
-|`bus_select`| int | **Required** | The index of the I<sup>2</sup>C bus. |
+|`bus`| string | **Required** | The index of the I<sup>2</sup>C bus. |
```

Comment:
```suggestion
|`bus`| string | **Required** | The index of the I<sup>2</sup>C bus. <br> Example: `"1"` |
```
(opt) Would help to put example on here-- assuming this is what it looks like if it's string

Reply:
I wasn't actually sure what a good example would be--would this example apply to all board models?

Reply:
I believe so but SME could confirm

Reply:
@randhid Is this example applicable to i2c buses on all boards? TY!

Reply:
Sounds like this doesn't matter since it's all about to change